qiita_ware/test/test_ebi.py

```diff
@@ -243,6 +244,27 @@ def test_generate_curl_command(self):
                '%%20ebi_access_key%%3D"') % {'xml_dir': submission.xml_dir}
         self.assertEqual(obs, exp)
 
+    def test_parse_EBI_reply(self):
+        e = EBISubmission(2, 'ADD')
+        curl_result = ""
```

Comment:
Yes, but we still don't have the DB to support such data - so I think it makes sense to leave as it is in this PR and modify once we modify the DB (PR in progress currently).